### ISSUE TITLE
Build throws error `ReferenceError: ssrRender is not defined`

### ISSUE THREAD

**Issue by marina-mosti** [vuejs/vitepress#12]
## Description
On a Vue 3 beta 12 + VitePress 0.1.1 project, trying to build with `vitepress build` crashes with the following error

## Output
```
$ vitepress build
vitepress v0.1.1
building client bundle...
building server bundle...
rendering pages...
build error:
 ReferenceError: ssrRender is not defined
    at Object.<anonymous> (/Users/marina/Developer/formvuelatte/node_modules/vitepress/lib/app/temp/_assets/index.js:621:22)
    at Module._compile (internal/modules/cjs/loader.js:1133:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)
    at Module.load (internal/modules/cjs/loader.js:977:32)
    at Function.Module._load (internal/modules/cjs/loader.js:877:14)
    at Module.require (internal/modules/cjs/loader.js:1019:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.renderPage (/Users/marina/Developer/formvuelatte/node_modules/vitepress/dist/build/render.js:12:27)
    at Object.build (/Users/marina/Developer/formvuelatte/node_modules/vitepress/dist/build/build.js:13:28)
✨  Done in 3.61s.
```

The line that seems to be crashing is line 12 in `node_modules/vitepress/dist/build/render.js`
`const { createApp } = require(path_1.default.join(config.tempDir, '_assets/index.js'));`

## Repo to reproduce
To reproduce you can check out: https://github.com/vuelidate/formvuelatte/tree/next

**Comment by marina-mosti** [comment]
Seems like this error was addressed with upgrade to `0.2.0` however, now we're having different errors.

When serving for dev:
```
$ vitepress
vitepress v0.2.0
vite v0.16.10
listening at http://localhost:3000

  Error: ENOENT: no such file or directory, stat '/Users/marina/Developer/formvuelatte/node_modules/vitepress/dist/client/app/index.html'
      at Object.statSync (fs.js:1035:3)
      at Object.statSync (/Users/marina/Developer/formvuelatte/node_modules/fs-extra/node_modules/graceful-fs/polyfills.js:308:16)
      at Object.cachedRead (/Users/marina/Developer/formvuelatte/node_modules/vite/dist/utils/fsUtils.js:20:45)
      at /Users/marina/Developer/formvuelatte/node_modules/vitepress/dist/node/server.js:98:30
      at async /Users/marina/Developer/formvuelatte/node_modules/vite/dist/server/serverPluginHtml.js:51:9
      at async /Users/marina/Developer/formvuelatte/node_modules/vite/dist/server/serverPluginModuleRewrite.js:27:9
```

And when building:
```
$ vitepress build
vitepress v0.2.0
vite v0.16.10
building client bundle...
build error:
 [Error: Could not load /Users/marina/Developer/formvuelatte/node_modules/vite/node_modules/@vue/shared/dist/shared.esm-bundler.js (imported by /Users/marina/Developer/formvuelatte/node_modules/vite/node_modules/@vue/runtime-dom/dist/runtime-dom.esm-bundler.js): ENOENT: no such file or directory, open '/Users/marina/Developer/formvuelatte/node_modules/vite/node_modules/@vue/shared/dist/shared.esm-bundler.js'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/marina/Developer/formvuelatte/node_modules/vite/node_modules/@vue/shared/dist/shared.esm-bundler.js',
  watchFiles: [
    '/Users/marina/Developer/formvuelatte/CHANGELOG.md',
    '/Users/marina/Developer/formvuelatte/README.md',
    '/Users/marina/Developer/formvuelatte/index.md',
    '/Users/marina/Developer/formvuelatte/README.md?vue&type=script&lang.js',
    '/Users/marina/Developer/formvuelatte/README.md?vue&type=template&id=4fa18e81',
    '/Users/marina/Developer/formvuelatte/node_modules/vite/node_modules/@vue/runtime-dom/dist/runtime-dom.esm-bundler.js',
    '/Users/marina/Developer/formvuelatte/node_modules/vitepress/dist/client/app/index.js',
    '/Users/marina/Developer/formvuelatte/CHANGELOG.md?vue&type=script&lang.js',
    '/Users/marina/Developer/formvuelatte/CHANGELOG.md?vue&type=template&id=0794ed8b',
    '/Users/marina/Developer/formvuelatte/index.md?vue&type=script&lang.js',
    '/Users/marina/Developer/formvuelatte/index.md?vue&type=template&id=1dec1726',
    '/Users/marina/Developer/formvuelatte/.vitepress/theme/index.js',
    'vite/hmr',
    '/Users/marina/Developer/formvuelatte/node_modules/vite/node_modules/@vue/runtime-core/dist/runtime-core.esm-bundler.js',
    '/Users/marina/Developer/formvuelatte/node_modules/vite/node_modules/@vue/shared/dist/shared.esm-bundler.js'
  ]
}
(node:6342) Warning: Accessing non-existent property 'lineno' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
(node:6342) Warning: Accessing non-existent property 'column' of module exports inside circular dependency
(node:6342) Warning: Accessing non-existent property 'filename' of module exports inside circular dependency
(node:6342) Warning: Accessing non-existent property 'lineno' of module exports inside circular dependency
(node:6342) Warning: Accessing non-existent property 'column' of module exports inside circular dependency
(node:6342) Warning: Accessing non-existent property 'filename' of module exports inside circular dependency
```

**Comment by marina-mosti** [comment]
Closing this since its now dupped by #15
